### ISSUE TITLE
Keyboard-only user cannot move open documents between source panes

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#7819]
### System details

    RStudio Edition : Desktop or Server
    RStudio Version : 1.4.833
    OS Version      : any
    R Version       : any

### Steps to reproduce the problem
- open some documents in IDE
- create a new source column ("Add Source Column")
- try to move a document from its current source pane to a different one using only the keyboard

### Describe the problem in detail
- cannot do this; have to use mouse to drag & drop the tab

### Describe the behavior you expected
- keyboard-only user has a way to do this

- [x] I have read the guide for [submitting good bug reports](https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports).
- [x] I have installed the latest version of RStudio, and confirmed that the issue still persists.
- [x] If I am reporting a RStudio crash, I have included a [diagnostics report](https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report).
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.
